ChangeLog

```diff
@@ -1,3 +1,15 @@
+Version 3.1.2
```

Comment:
Does this project follow SemVer? Is it really appropriate to only increment a patch version if you're dropping support for Python 3.5?

Reply:
`3.2.0` may make more sense, but let's wait for saghul's opinion.

Reply:
Yep, let's bump the minor indeed.

Reply:
I've updated to 3.2.0.